src/style.css

```diff
@@ -1,3 +1,92 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+a.img {
+  display: block;
+  position: relative;
+  width: 40px;
+  height: 40px;
+  top: 10px;
+}
+
+a.blue {
+  color: var(--main-color);
+}
+
+.container {
+  display: flex;
+  font-size: 12px;
+}
+
+.header {
```

Comment:
The `.header` class is defined here, but in the HTML, the header uses the class `header` without a dot. Ensure consistency between the HTML and CSS class names.